common/translations/lang/en.json

```diff
@@ -193,10 +222,12 @@
     "BULK_Label_1": "Number of Wallets To Generate ",
     "BULK_Label_2": "Generate Wallets ",
     "BULK_SuccessMsg": "Success! Your wallets have been generated. ",
-    "SEND_addr": "To Address ",
+    "SEND_addr": "To ",
```

Comment:
We should add a `SEND_addr_short` instead, I think we may want to keep this key around.